biothings/utils/es.py

```diff
@@ -950,7 +955,57 @@ def set_internal_number_of_replicas(self, number_of_replicas=None):
         if not number_of_replicas:
             number_of_replicas = self.number_of_replicas
         settings = json.dumps({"index": {"number_of_replicas": number_of_replicas}})
-        self._es.indices.put_settings(settings, index=self._index)
+        self.update_settings(settings)
+
+    def update_settings(self, settings, remove_meta_fields=False, close=False, **params):
+        if remove_meta_fields:
```

Comment:
@sengineer0 please add doc about this new method and its parameters. 

- `remove_meta_fields`: are there use cases where these settings fields should be kept? Except for `number_of_shards`, should other fields never be included? And `number_of_shards` should be a valid setting field, why do we remove it?
- `close` parameter: why it's needed here?

Reply:
I removed `remove_meta_fields` from `update_settings` method.